public/css/signup.css

```diff
@@ -0,0 +1,278 @@
+body {
+    background-color: #dfdfdf;
+}
+
+/* Universal Selector */
+* {
+    
```

Comment:
There's a bunch of duplicate code between `login.css` and `signup.css`, maybe run them through a diff tool and factor out the common lines into a `common.css` file?

Reply:
Used style.css for the common code.